react-view/src/pages/MultiWidgetPage/widgets/BaseWidget.tsx

```
@@ -0,0 +1,23 @@
+import React from "react";
+
+type BaseState = {
+    widgetType: string;
+    viewType: string;
+    layoutParams: string;
+}
+
+export abstract class BaseWidget<T extends BaseState> extends React.Component<any, T> {
+  public constructor(props: any) {
+    super(props);
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)